src/styles/youtubeMockup.scss

```diff
@@ -0,0 +1,295 @@
+    display: none;
+  }
+}
+
+.toolbar-icons {
+  padding: 10px;
+}
+
+.toolbar-icons-end {
+  padding: 10px;
+  @media (max-width: 425px) {
+    display: none;
+  }
+}
+.tabs-boxed {
+  background-color: transparent !important;
+}
+
+.tabs {
+  overflow: hidden;
+}
+
+//browser navbar styles
+.mockup-navbar,
+.youtube-mockup-navbar {
+  display: flex;
+  align-items: center;
+  min-height: 3rem;
+  width: 100%;
+  background-color: $blackOne;
+  button {
+    padding: 5px;
+  }
+}
+
+.youtube-mockup-navbar {
+  background-color: $grayOne;
+}
+
+.mockup-navbar-start,
+.youtube-navbar-start {
+  display: flex;
+  width: 100%;
+  justify-content: flex-start;
+  padding-left: 5px;
+  @media (max-width: 320px) {
+    display: none;
+  }
+}
+
+.mockup-navbar-start {
+  @media (max-width: 600px) {
+    display: none;
+  }
+}
+
+.mockup-navbar-end {
+  @media (max-width: 640px) {
+    display: none;
+  }
+}
+
+.mockup-navbar-end,
+.youtube-navbar-end {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+  padding-right: 5px;
+}
+
+.mockup-navbar-end .skeleton,
+.mockup-navbar-end .toolbar-icons {
+  @media (max-width: 640px) {
+    display: none;
+  }
+}
+
+.youtube-navbar-end {
+  @media (max-width: 640px) {
+    visibility: hidden;
+  }
+}
+
+.mockup-navbar-middle {
+  flex: 1;
+  display: flex;
+  justify-content: center;
+
+  @media (max-width: 375px) {
+    display: left;
+    width: 50%;
+    padding-left: 5px;
+  }
+
+  @media (max-width: 520px) {
+    display: left;
+    width: 80%;
+    padding-left: 5px;
+  }
+}
+
+.address-bar {
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  width: 100%;
+  height: 2rem;
+  padding: 0 1rem;
+  font-size: 1rem;
+  line-height: 1.5rem;
+  border-radius: 0.5rem;
+  border: 1px solid transparent;
+  background-color: $grayTwo;
+  color: $whiteOne;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+//youtube
+.youtube-search-wrapper {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  @media (max-width: 320px) {
+    padding-left: 20px;
+  }
+}
+
+.youtube-search {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  background-color: $blackOne;
+  border-radius: 0.5rem;
+  border-color: $youtubeBorder;
+  border-width: 2px;
+}
+
+.youtube-search-input {
+  background: transparent;
+}
+
+.youtube-search-icon {
+  background-color: $youtubeBorder;
+}
+
+.rounded-icon {
+  background-color: $youtubeBorder;
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  margin-left: 10px;
+}
+.youtube-microphone-icon {
+  background-color: $youtubeBorder;
+  width: 30px;
+  height: 30px;
+  border-radius: 50%;
+  margin-left: 10px;
+}
+
+.skeleton {
+  border-color: $youtubeBorder;
+  border-width: 1px;
+  background-color: $grayOne;
+  background-image: linear-gradient(
+    105deg,
+    transparent 0%,
+    transparent 40%,
+    $blackOne 50%,
+    transparent 60%,
+    transparent 100%
+  );
+}
+
+.content-wrapper {
```

Comment:
I would be careful with generic class names like this that you don't have scoped within other classes. This could override other classes, especially framework classes pretty easily.